ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/entity/QueryDescription.java

```diff
@@ -42,7 +45,9 @@
   private final String executionPlan;
   private final Map<String, Object> overriddenProperties;
   private final Optional<String> state;
+  private final TreeMap<KsqlHostInfoEntity, String> ksqlHostQueryState;
```

Comment:
nit:

```suggestion
  private final Map<KsqlHostInfoEntity, String> ksqlHostQueryState;
```

Is sufficient, no?